lib/components/tab_bar/tab_bar.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:tarka_ui/styles/theme.dart';
+
+import '../button/button.dart';
+import '../button/style.dart';
+
+class TUITabBar extends StatelessWidget {
+  final List<String> titles;
+  final int selectedIndex;
+  final Function(int)? onPressed;
+
+  const TUITabBar({
+    super.key,
+    required this.titles,
+    required this.selectedIndex,
+    required this.onPressed,
+  });
+
+  Widget _buildItem(int index, String item) {
+    return TUIButton(
```

Comment:
For regular size, we have to use a 32px sized button, and for the large one, we have to use a 40px sized button.